### PR TITLE
Various improvements for PM query editor

### PR DESCRIPTION
**Pull request by brynjare**
Currently no Jira issue for this pull request, since I wasn't sure which Issue Type to select in the absence of `Enhancement`

This is a couple of minor commits to add what I hope will be considered as improvements to the query editor in the Performance Management datasource. The branch is based on the (currently) last commit in #43.

### CODE REVIEW

**Comment by mvrueden** [comment]
Some of these changes seem to conflict with HELM-71.

**Comment by brynjare** [comment]
I probably broke some best practices when I submitted my pull request. It already includes all the commits from HELM-71, since I based my further changes on those fixes.

Sorry for the trouble! Let me know if I need to change something or create a new pull request.

**Comment by mvrueden** [comment]
In general this looks good to me. 
However, @j-white did most of the work on this datasource. @j-white could you review this?! (-:

File: src/datasources/perf-ds/query_help.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#### Names and labels
```
[the remark this answers is not in the document]
Comment (brynjare) [reply src/datasources/perf-ds/query_help.md]: I don't really mind either way. I just realised that the old [inline help](https://github.com/OpenNMS/opennms-helm/blob/4c218a44bd87c8100287de06fc4ddb9b973f1777/src/datasources/perf-ds/partials/query.options.html#L43-L90) was deprecated by Grafana at some point, and even though the help text is still in opennms-helm, it's no longer accessible.

By creating this file, with a markdown-version of the text, Grafana automatically includes a "Help"-tab for each query:

query-help

The current help text could be considered more of a PoC or a placeholder than anything, I suppose.

**Comment by brynjare** [comment]
Thanks for the feedback so far! No worries about any delays; I know how it is. I'm glad to hear opennms-helm is still alive, though, because it's been extremely useful so far.

I do agree that this probably should never have been a single pull request. I just got a bit carried away during what was supposed to be a simple fix. :)

Due to working on other projects these days, I'm not sure when I'll be able to split it up into smaller PRs, but you're free to close this in the meantime. (Or cherry-pick / adapt the current commits as you see fit, for that matter.)

**Comment by brynjare** [comment]
Now that HELM 2.0 is released, I'll close this pull request and focus on creating proper issues in JIRA for any bugs / features still relevant in the current release. Sorry about messing this one up. :)

**Pull request closed without merging**